app/routes/how-can-i-help.knowledge.tsx

```diff
@@ -79,10 +85,97 @@ const NewToAISafety = () => (
 const Dropdowns = () => (
   <div className="padding-bottom-80">
     <DropDown title="Books are more my thing">
-      <p>TODO fill this</p>
+      <HelpItem
+        links={[
+          {
+            tag: 'Top recommendation',
+            title: 'Uncontrollable',
+            action: 'https://www.goodreads.com/book/show/202416160-uncontrollable',
+          },
+        ]}
+        additionalInfo={
+          <>
+            Or, read our{' '}
+            <a href="https://aisafety.info/questions/8159" className="small-bold teal-500">
+              full list of AI safety book recommendations
+            </a>
+          </>
+        }
+      >
+        We recommend the book "Uncontrollable," which concisely examines the risks posed by advanced
+        AI. The book highlights the need for effective AI governance and safety measures, and offers
+        practical solutions to ensure AI benefits society while minimizing risks.
+      </HelpItem>
     </DropDown>
-    <DropDown title="I want something I can subscribe to, so I can learn more about AI safety consistently">
-      <p>TODO fill this</p>
+
+    <DropDown title="I want something I can subscribe to (Youtube channels, newsletters, etc.)">
+      <HelpItem
+        title="Youtube"
+        links={[
+          {
+            tag: 'Top recommendation',
+            title: 'Rob Miles AI Safety',
```

Comment:
"Robert"

Reply:
I don't know if Rob goes by "Robert" officially, but I changed all instances of "Rob" to "Robert"

Reply:
His yt channel is "Robert" but everywhere else I've seen/heard "Rob"

Reply:
Yeah I think it's just his YouTube. His Discord server for example is Rob. I would probably refer to him as Rob everywhere except when naming his YouTube